src/components/contributors.json

```diff
@@ -41,5 +41,13 @@
     "avatar": "rahul.png",
     "city": "Hyderabad",
     "quote": "Change things"
+  },
+  {
+    "github": "mathpalvipin",
+    "name": "Vipin Mathpal",
+    "organization": "....",
```

Comment:
you can completely remove the organization field if you don't have one.
Also, you can add the college field if you want.